include/deal.II/grid/grid_tools.h

```diff
@@ -3286,69 +3286,6 @@ namespace GridTools
   compute_vertices_with_ghost_neighbors(
     const Triangulation<dim, spacedim> &tria);
 
-  /**
-   * A structure that allows the transfer of cell data of type @p T from one processor
-   * to another. It corresponds to a packed buffer that stores a vector of
-   * CellId and a vector of type @p T.
-   *
-   * This class facilitates the transfer by providing the save/load functions
-   * that are able to pack up the vector of CellId's and the associated
-   * data of type @p T into a stream.
-   *
-   * Type @p T is assumed to be serializable by <code>boost::serialization</code> (for
-   * example <code>unsigned int</code> or <code>std::vector@<double@></code>).
-   */
-  template <int dim, typename T>
-  struct CellDataTransferBuffer
```

Comment:
Shouldn't we deprecate this class first?

Reply:
That's exactly what I wanted to ask here, I am unsure. But we never use it (now), we do not test it explicitly, and have no other reference to it.